worlds/stardew_valley/__init__.py

```diff
@@ -118,36 +99,22 @@ def create_region(name: str, exits: Iterable[str]) -> Region:
 
         world_regions, world_entrances, self.randomized_entrances = create_regions(create_region, self.random, self.options)
 
-        self.logic = StardewLogic(self.player, self.options, world_regions.keys())
-        self.modified_bundles = get_all_bundles(self.random,
-                                                self.logic,
-                                                self.options)
-
         def add_location(name: str, code: Optional[int], region: str):
             region = world_regions[region]
             location = StardewLocation(self.player, name, code, region)
             region.locations.append(location)
 
-        create_locations(add_location, self.modified_bundles, self.options, self.random)
+        create_locations(add_location, self.options, self.random)
```

Comment:
This is too complicated can you simplify this more? Maybe just don't create any locations?

Reply:
I wanted to keep all the features intact so users don't necessarily notice the difference, so all the locations must be the same